src/components/HeatMap.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Chart from 'react-apexcharts';
+// import ApexCharts from 'apexcharts';
+import axios from 'axios';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'axios'.